server/chat-plugins/wifi.tsx

```diff
@@ -733,18 +765,35 @@ export class GTS extends Rooms.RoomGame {
 	generateWindow() {
 		const sentModifier = this.sent.length ? 5 : 0;
 		const rightSide = this.noDeposits ?
-			`<strong>More Pokémon have been deposited than there are prizes in this giveaway and new deposits will not be accepted.
-			If you have already deposited a Pokémon, please be patient, and do not withdraw your Pokémon.</strong>
-			` :
-			`To participate, deposit <strong>${this.deposit}</strong> into the GTS and look for <strong>${Utils.escapeHTML(this.lookfor)}</strong>`;
-		return `<p style="text-align:center;font-size:14pt;font-weight:bold;margin-bottom:2px;">There is a GTS giveaway going on!</p>` +
-			`<p style="text-align:center;font-size:10pt;margin-top:0px;">Hosted by: ${Utils.escapeHTML(this.giver.name)} | Left: <b>${this.left}</b></p>` +
-			`<table style="margin-left:auto;margin-right:auto;"><tr>` +
-			(sentModifier ?
-				`<td style="text-align:center;width:10%"><b>Last winners:</b><br/>${this.sent.join('<br/>')}</td>` :
-				'') +
-			`<td style="text-align:center;width:15%">${this.sprite}</td><td style="text-align:center;width:${40 - sentModifier}%">${Chat.formatText(this.summary, true)}</td>` +
-			`<td style="text-align:center;width:${35 - sentModifier}%">${rightSide}</td></tr></table>`;
+			<strong>
+				More Pok&eacute;mon have been deposited than there are prizes in this giveaway and new deposits will not be accepted. {}
+				If you have already deposited a Pok&eacute;mon, please be patient, and do not withdraw your Pok&eacute;mon.
+			</strong> : <>
+				To participate, deposit <strong>{this.deposit}</strong> into the GTS and look for <strong>
+					{Utils.escapeHTML(this.lookfor)}
+				</strong>
```

Comment:
```suggestion
				To participate, deposit <strong>{this.deposit}</strong> into the GTS and look for <strong>{this.lookfor}</strong>
```

I'm guessing the double-escape was unintentional?